ansible/roles/operator-pipeline/templates/openshift/tasks/request-signature.yml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: request-signature
+spec:
+  params:
+    - name: manifest_digest
+      description: Manifest digest for the signed content, usually in the format sha256:xxx
+    - name: reference
+      description: Docker reference for the signed content, e.g. registry.redhat.io/redhat/community-operator-index:v4.9
+    - name: requester
+      description: Name of the user that requested the signing, for auditing purposes
+    - name: sig_key_id
+      description: The signing key id that the content is signed with
+      default: "4096R/55A34A82 SHA-256"
+    - name: sig_key_name
+      description: The signing key name that the content is signed with
+      default: containerisvsign
+    - name: umb_ssl_secret_name
+      description: Kubernetes secret name that contains the umb SSL files
+    - name: umb_ssl_cert_secret_key
+      description: The key within the Kubernetes secret that contains the umb SSL cert.
+    - name: umb_ssl_key_secret_key
+      description: The key within the Kubernetes secret that contains the umb SSL key.
+    - name: umb_client_name
+      description: Client name to connect to umb, usually a service account name
+      default: operatorpipelines
+    - name: umb_listen_topic
+      description: umb topic to listen to for responses with signed content
+      default: VirtualTopic.eng.robosignatory.isv.sign
+    - name: umb_publish_topic
+      description: umb topic to publish to for requesting signing
+      default: VirtualTopic.eng.operatorpipelines.isv.sign
+    - name: umb_url
+      description: umb host to connect to for messaging
+      default: umb.api.redhat.com
+  results:
```

Comment:
Can you extend results to return values that are needed by upload-signature task?

Currently `repository` and `sig_key_id` are missing.

Reply:
`repository` wasn't actually used by the signing request, so I removed it from the params. I can add `sig_key_id`, but that's gonna need to be passed in from the pipeline based on environments (pre-prod should use the redhate2etesting key, key id 4096R/37036783 SHA-256

Reply:
If it makes things easier, I could set the sig_key_* within the request signing script instead, and take in an env param. I wanted to avoid hardcoding, but that's what RHC Cert Service does currently, and those values are very very unlikely to change.

Reply:
If it requires something that depends on the environment you can add it into `set-env` task and I can re-use the value in the new pipeline and pass it to your task as an argument.

Reply:
Good point. Added all the env-dependent values to the `set-env` task in the latest commit.